README.bazel.md

```diff
@@ -88,11 +102,12 @@ Then edit the new entry in `tools/BUILD` to replace `package_name_snake_case` wi
 
 After adding a new `import` statement in Go code, run `bazel configure` to update the BUILD file.
 
-If the package is not already a dependency of the project, you'll have to do some additional steps:
+If the package is not already a dependency of the project, you'll have to do some additional steps.
+Run these commands from the workspace root:
 
 ```shell
 # Update go.mod and go.sum, using same Go SDK as Bazel
-% bazel run @rules_go//go -- mod tidy -v
+% ./tools/go mod tidy -v
```

Comment:
`$(bazel info workspace)/tools/go` to be consistent?

Reply:
SGTM. I don't think anyone would actually type that, but it's nice to be robust when copy-paste the command in a subfolder.